caep-interoperability-profile-1_0.md

```diff
@@ -0,0 +1,191 @@
+The `delivery` field MUST be present in the Configuration of any Stream generated by the Transmitter, and its value MUST include the following:
+
+~~~ json
+{
+    "method" : "urn:ietf:rfc:8935"
+}
+~~~
+
+#### Stream Control
+The following Stream Configuration API Methods MUST be supported:
+
+**Creating a Stream**
+: Receivers MUST be able to create a Stream with the Transmitter using valid authorization with the Transmitter. The Transmitter MAY support multiple streams with the same Receiver
+
+**Reading Stream Configuration**
+: A Receiver MUST be able to obtain current Stream configuration from the Transmitter by providing a valid authorization
+
+**Getting the Stream Status**
+: A Receiver MUST be able to obtain the current Stream status from the Transmitter by providing a valid authorization
+
+**Stream Verification**
+: A Receiver MUST be able to verify the liveness of the Stream by requesting that the Transmitter send it a Stream Verificaiton event
+
+## Receivers {#common-receivers}
+Receivers MUST implement the following features:
+
+### Push Delivery {#common-receiver-push}
+Receivers MUST be able to accept events using the Push-Based Security Event Token (SET) Delivery Using HTTP {{RFC8935}} specification.
+
+### Implicitly Added Subjects {#common-receiver-subjects}
+Receivers MUST assume that all subjects are implicitly included in a Stream, without any `AddSubject` method invocations.
+
+## Event Subjects {#common-event-subjects}
+The following subject identifier formats from "Subject Identifiers for Security Event Tokens" {{RFC9493}} MUST be supported:
+
+* `email`
+* `iss_sub`
+
+Receivers MUST be prepared to accept events with any of the subject identifier formats specified in this section. Transmitters MUST be able to send events with at least one of subject identifier formats specified in this section.
+
+## Event Signatures
+All events MUST be signed using the `RS256` algorithm using a minimum of 2048-bit keys.
+
+# Use Cases
+Implementations MAY choose to support one or more of the following use-cases in order to be considered interoperable implementations
+
+## Session Revocation / Logout
+In order to support session revocation or logout, implementations MUST support the CAEP event type `session-revoked`. The `reason_admin` field of the event MUST be populated with a non-empty value.
+
+## Credential Change
+In order to support notifying and responding to credential changes, implementations MUST support the CAEP event type `credential-change`.
+Within the `credential-change` event, implementations MUST support the following field values:
+
+`change_type`
+: Receivers MUST interpret all allowable values of this field. Transmitters MAY generate any allowable value of this field
+
+`credential_type`
+: Receivers MUST interpret the following values of this field. Transmitters MAY generate any of the following values:
+
+  * `password`
+  * `pin`
+  * `fido2-platform`
+  * `fido2-roaming`
+  * `fido2-u2f`
```

Comment:
We should include all the existing values that exist in the CAEP spec

Reply:
Since it is a "MUST" include, I feel we can add a few more like "phone-voice" and "phone-sms" can be added, but I don't want to require Transmitters and Receivers to have to support all of them.

Reply:
As a general comment, I think it is challenging to only support a subset of an event type. In my opinion, the most specific requirement should be the event type holistically, not certain parts of it.

A transmitter should only be expected to support the properties that is supports in its respective platform. Since we're not telling receivers what to do with a given property, they should only be required to validate that the property is present and of the correct format.